RecoTauTag/HLTProducers/src/PFJetsTauCorrelationCondition.cc

```diff
@@ -0,0 +1,86 @@
+#include "RecoTauTag/HLTProducers/interface/PFJetsTauCorrelationCondition.h"
+#include "Math/GenVector/VectorUtil.h"
+#include "DataFormats/HLTReco/interface/TriggerTypeDefs.h"
+#include "FWCore/Utilities/interface/EDMException.h"
+#include "DataFormats/TauReco/interface/PFTau.h"
+#include "DataFormats/Math/interface/deltaR.h"
+
+//
+// class declaration
+//
+PFJetsTauCorrelationCondition::PFJetsTauCorrelationCondition(const edm::ParameterSet& iConfig)
+    : tauSrc_(consumes<trigger::TriggerFilterObjectWithRefs>(iConfig.getParameter<edm::InputTag>("TauSrc"))),
+      pfJetSrc_(consumes<reco::PFJetCollection>(iConfig.getParameter<edm::InputTag>("PFJetSrc"))),
+      extraTauPtCut_(iConfig.getParameter<double>("extraTauPtCut")),
+      mjjMin_(iConfig.getParameter<double>("mjjMin")),
+      matchingR2_(iConfig.getParameter<double>("Min_dR") * iConfig.getParameter<double>("Min_dR")) {
+    produces<reco::PFJetCollection>();
+}
+PFJetsTauCorrelationCondition::~PFJetsTauCorrelationCondition() {}
+
+void PFJetsTauCorrelationCondition::produce(edm::StreamID iSId, edm::Event& iEvent, const edm::EventSetup& iES) const {
+    std::unique_ptr<reco::PFJetCollection> cleanedPFJets(new reco::PFJetCollection);
+
+    edm::Handle<trigger::TriggerFilterObjectWithRefs> tauJets;
+    iEvent.getByToken(tauSrc_, tauJets);
```

Comment:
The `tauJets` handle is not needed, see comment to ll. 30-31. So ll. 24-25 can be removed.